locales/lt-LT/messages.po

```diff
@@ -0,0 +1,1007 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: TAO 3.4.0-sprint112\n"
+"PO-Revision-Date: 2019-10-09T09:03:14\n"
+"Last-Translator: TAO Translation Team <translation@tao.lu>\n"
+"MIME-Version: 1.0\n"
+"Language: lt-LT\n"
+"sourceLanguage: en-US\n"
+"targetLanguage: lt-LT\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Poedit-Basepath: ../../\n"
+"X-Poedit-KeywordsList: __\n"
+"X-Poedit-SearchPath-0: .\n"
+
+msgid "%1$s - Item %2$s/%3$s"
+msgstr "%1$s - Elementas %2$s/%3$s"
+
+msgid "%d sessions of the delivery %s have not been paused"
+msgstr "%d pateikties sesijos %s nebuvo pristabdytos"
+
+msgid "%s - item %p/%c"
+msgstr ""
+
+msgid "%s completed"
+msgstr "%s atlikta"
+
+msgid "%s could not be %s because it is finished. Please refresh your data."
+msgstr "%s negali būti %s, nes jis yra užbaigtas. Prašome atnaujinti savo duomenis."
+
+msgid "%s could not be %s because it is terminated. Please refresh your data."
+msgstr "%s negali būti %s, nes jis yra nutrauktas. Prašome atnaujinti savo duomenis."
+
+msgid "%s could not be %s."
+msgstr "%s negali būti %s."
+
+msgid "%s minutes more"
+msgstr "%s minučių daugiau"
+
+msgid "%s\'s test had already been authorized."
+msgstr ""
+
+msgid "%s\'s test had already been paused."
+msgstr ""
+
+msgid "%s\'s test had already been terminated."
```

Comment:
As I remember it will not work if you use backslashes

Reply:
yes, you right. it fixed